### PR TITLE
addWorkerLoader

### PR DESCRIPTION
**Pull request by liyuanqiu**

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -240,6 +240,26 @@ const addLessLoader = (loaderOptions = {}) => config => {
   return config;
 };
 
+// Use this helper to add `worker-loader` into webpack loaders.
+const addWorkerLoader = (loaderOptions = {}) => config => {
+  const loaders = config.module.rules.find(rule => Array.isArray(rule.oneOf)).oneOf;
+
+  loaders.splice(loaders.length - 1, 0, {
```
Comment (FezVrasta) [inline index.js]: What is this splice doing here? Adding to the end of the array the new object?

Comment (liyuanqiu) [reply index.js]: Keep the same behavior with `addLessLoader`.

```javascript
  // Insert less-loader as the penultimate item of loaders (before file-loader)
  loaders.splice(loaders.length - 1, 0, {
    test: lessRegex,
    exclude: lessModuleRegex,
```

Comment (liyuanqiu) [reply index.js]: I'll give an online example later.

Comment (FezVrasta) [reply index.js]: Oh ok I see, thanks for the explanation!

Comment (liyuanqiu) [reply index.js]: Sorry, I found some error to fix. You are right, the loader shouldn't be added as the penultimate item. This merge should be reverted.

Here is an example:
https://github.com/liyuanqiu/addWorkerLoaderDemo

**Pull request merged**